lua/telescope/_extensions/zk.lua

```diff
@@ -0,0 +1,135 @@
+local pickers = require("telescope.pickers")
+local finders = require("telescope.finders")
+local conf = require("telescope.config").values
+local actions = require("telescope.actions")
+local action_state = require("telescope.actions.state")
+local action_utils = require("telescope.actions.utils")
+local putils = require("telescope.previewers.utils")
+local entry_display = require("telescope.pickers.entry_display")
+local previewers = require("telescope.previewers")
+
+local function request_notes(bufnr, options, cb)
+  vim.lsp.buf_request(bufnr, "workspace/executeCommand", {
+    command = "zk.list",
+    arguments = {
+      vim.api.nvim_buf_get_name(bufnr),
+      vim.tbl_extend("error", { select = { "title", "absPath", "rawContent" } }, options or {}),
+    },
+  }, function(err, res)
+    assert(not err, err)
```

Comment:
```suggestion
    assert(not err, tostring(err))
```
in my experience, `err` was a table and needed to be converted to a string.